### DISCUSSION TITLE
about contributing

### DISCUSSION THREAD

**Discussion by Kalmaegi** [zed-industries/zed#7060] [Feature Requests]
Sorry to bother you, I would like to submit some PR to zed, in the [top-ranking issues](https://github.com/zed-industries/zed/issues/5393) if I choose an item that can be submitted, how do I need to confirm that I have claimed the task, I am just worried that I and others have modified the same issue at the same time, causing conflicts

**Comment by mrnugget** [comment]
Hey! You can just leave a comment in the issue saying that you start working on this (and should you stop, you can also let others know so they might pick it up).

**Comment by osiewicz** [comment]
Hey there, thanks for reaching out. :)
At the moment it's still a bit fuzzy. If the PR you intent to make is small, just feel free to open it and mention the issue that it should close (with syntax like `Fixes #7060`). If you expect to need a bit more time, just drop a comment.

**Comment by mikayla-maki** [comment]
Also! If you're going to pick up something that's more complex, I highly recommend finding time to pair with us on the initial design. Either at our fireside hacks, or we schedule something directly. We can point you to the right files, talk through design issues, make sure it'll be fast, etc. etc. :D

**Comment by Kalmaegi** [comment]
Thank you very much for your patience and I'd like to start by fixing some known issues, such as this one: [Removing empty brackets #4605
](https://github.com/zed-industries/zed/issues/4605)
